modules/vectorfieldvisualizationgl/src/processors/2d/lic2d.cpp

```diff
@@ -68,37 +87,61 @@ LIC2D::LIC2D()
                         false}
     , brightness_{"brightness",
                   "Brightness",
-                  "Adjusts the overall brightness"_help,
+                  "Adjusts the overall brightness."_help,
                   0.0f,
                   {-1.0f, ConstraintBehavior::Immutable},
                   {1.0f, ConstraintBehavior::Immutable}}
     , contrast_{"contrast",
                 "Contrast",
-                "Adjusts the overall contrast"_help,
+                "Adjusts the overall contrast."_help,
                 0.0f,
                 {-1.0f, ConstraintBehavior::Immutable},
                 {1.0f, ConstraintBehavior::Immutable}}
     , gamma_{"gamma",
              "Gamma Correction",
-             "Gamma correction using $v^gamma$"_help,
+             "Gamma correction using $v^gamma$."_help,
```

Comment:
don't think we handle inline latex anymore. since we are not passing this through doxygen anymore :(